GSLabMisc/man/LoadScalars.Rd

```diff
@@ -0,0 +1,20 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/LoadScalars.R
+\name{LoadScalars}
+\alias{LoadScalars}
+\title{Load scalars from a text file into global environment.}
+\usage{
+LoadScalars(input, where = 1, convert = TRUE, ...)
+}
+\arguments{
+\item{input}{path to \code{.txt} file.}
+
+\item{where}{where to load the scalars. Defaults to 1 to load to the global environment.}
```

Comment:
Are we sure we want this option given that the title of the package is "Load scalars...into global environment"?

Reply:
@jmshapir That's a good point. Do you have a preference between changing the title of the package and getting rid of the option?

Also in terms of naming, is it OK that the function is called LoadScalars when it can also load 1-dimensional arrays?

Reply:
@ew487 thanks!

> @jmshapir That's a good point. Do you have a preference between changing the title of the package and getting rid of the option?

My instinct is to drop the option.

> 
> Also in terms of naming, is it OK that the function is called LoadScalars when it can also load 1-dimensional arrays?

I'm ok with the name LoadScalars. But if we drop the option per above, we could alternatively call the function LoadGlobals, which would be more descriptive and wouldn't imply a restriction to scalars.

Reply:
@jmshapir Sounds good, I will drop it then.

Reply:
Fixed in [a384847](https://github.com/gslab-econ/gslab_r/pull/26/commits/a3848479dd05f87ef4f384c8cfffe06c9c751c14)

Reply:
Thanks @ew487! Do you think it would be clearer to call the function `LoadGlobals` now that we're enforcing that objects are loaded into the global environment?

Reply:
@jmshapir OK, I think that sounds good. I just changed it in [601e953](https://github.com/gslab-econ/gslab_r/pull/26/commits/601e953eab79d24b6ab3ff5ddebfab3d72b94c5f)